wai-saml2.cabal

```
@@ -1,6 +1,6 @@
 cabal-version: 1.12
 
--- This file has been generated from package.yaml by hpack version 0.35.0.
+-- This file has been generated from package.yaml by hpack version 0.34.7.
```

It would be good if this could stay the same / use the `hpack` that is bundled with the latest version of `stack`.
```suggestion
-- This file has been generated from package.yaml by hpack version 0.35.0.
```